vm/rust/src/class.rs

```diff
@@ -1,21 +1,21 @@
+use starknet::core::types::contract::legacy::LegacyContractClass;
 use std::{
     ffi::{c_char, c_uchar, CStr},
     slice,
 };
-use starknet::core::types::contract::legacy::LegacyContractClass;
 
 #[no_mangle]
 pub extern "C" fn Cairo0ClassHash(class_json_str: *const c_char, hash: *mut c_uchar) {
     let class_json = unsafe { CStr::from_ptr(class_json_str) }.to_str().unwrap();
     let class: Result<LegacyContractClass, serde_json::Error> = serde_json::from_str(class_json);
     if class.is_err() {
-        return
+        return;
     }
     let class_hash = class.unwrap().class_hash();
     if class_hash.is_err() {
-        return
+        return;
```

Comment:
is there a difference with `;` ?

Reply:
Functionally, probably not. It is just a formatting/style thing. 

It was `cargo fmt` who changed it.